web/client/utils/openlayers/DrawSupportUtils.js

```diff
@@ -31,7 +34,7 @@ export const transformPolygonToCircle = (feature, mapCrs) => {
         } else {
             center = getCenter(extent);
         }
-        const radius = feature.getProperties().radius || calculateRadius(center, feature.getGeometry().getCoordinates());
```

Comment:
Hi, why have you removed `feature.getProperties().radius` ?

Reply:
Because when map projection changes that radius value is no longer valid